test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/TestSuitInitializer.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common
+{
+	/* Responsible for all initilisation before actual test startup -
+     * Creation of Azure resources - Eventhubs and Storage Queues
+     * Function App startup
```

Comment:
Format in this and other files too

Reply:
Was due to inconsistent use of tabs and spaces. Corrected.